airbyte-scheduler/client/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+    id "java-library"
+}
+
+dependencies {
+    implementation project(':airbyte-config:models')
+    implementation project(':airbyte-protocol:models')
+    implementation project(':airbyte-scheduler:models')
+    implementation project(':airbyte-scheduler:persistence')
+    // todo (cgardens) - remove this dep. just needs temporal client.
```

Comment:
should this be handled now?

Reply:
unfortunately no. in a subsequent PR I will break out a temporal client, but this one is big enough 😄